### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by Ant1r**
note: this PR contains some fixes stolen from another one #319 from jpcima (sorry I should have merged instead but remembered it too late). So IMO #319 could be closed; I add jpcima to the french translators list.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -623,7 +623,11 @@ msgstr "[deken] deken-plugin.tcl (recherche d'externals) chargé depuis %s."
 
 #, tcl-format
 msgid "[deken] Platform detected: %s"
-msgstr "[deken] Platforme detectée : %s"
+msgstr "[deken] Platforme détectée : %s"
```
Comment (trebmuh) [inline po/fr.po]: Should be "Plateforme"

File: po/fr.po [hunk 2]
```diff
@@ -623,7 +623,11 @@ msgstr "[deken] deken-plugin.tcl (recherche d'externals) chargé depuis %s."
...
+
+#, tcl-format
+msgid "[deken] Platform re-detected: %s"
+msgstr "[deken] Platforme re-détectée : %s"
```
Comment (trebmuh) [inline po/fr.po]: Should be "Plateforme" too

**Comment by umlaeute** [comment]
merged into update/0.48-2

**Comment by trebmuh** [comment]
:+1:

**Pull request closed without merging**